packages/sdk/src/libs/clients/lsp/client.ts

```diff
@@ -0,0 +1,103 @@
+import { LongShortPair__factory, LongShortPair } from "@uma/core/contract-types/ethers";
+import type { SignerOrProvider, GetEventType } from "../..";
+import { Event } from "ethers";
+import { Balances } from "../../utils";
+
+export type Instance = LongShortPair;
+
+export type TokensCreated = GetEventType<Instance, "TokensCreated">;
+export type TokensRedeemed = GetEventType<Instance, "TokensRedeemed">;
+export type ContractExpired = GetEventType<Instance, "ContractExpired">;
+export type PositionSettled = GetEventType<Instance, "PositionSettled">;
```

Comment:
yea its possible, but by keeping naming consistent between all clients, we kind of form a convention (in every client file we can reference Instance and not worry about naming) which then makes it really easy to understand and add new clients.

Reply:
Sounds good! I would still argue that it's a bit harder to understand as a reader since we're removing context about the type that these events apply to, but I think that's fine.